SPECS/apache-commons-cli/apache-commons-cli.spec

```diff
@@ -22,12 +22,12 @@ Distribution:   Mariner
 %global short_name      commons-%{base_name}
 Name:           apache-commons-cli
 Version:        1.4
-Release:        4%{?dist}
+Release:        5%{?dist}
 Summary:        Command Line Interface Library for Java
 License:        Apache-2.0
 Group:          Development/Libraries/Java
 URL:            http://commons.apache.org/%{base_name}/
-Source0:        http://www.apache.org/dist/commons/%{base_name}/source/%{short_name}-%{version}-src.tar.gz
+Source0:        http://archive.apache.org/dist/commons/%{base_name}/source/commons-cli-%{version}-src.tar.gz
```

Comment:
Let's also lint/clean-up this spec. It looks like we didn't do it when the spec was added. At least that's what I'm getting from the order of the `Summary` and `Distribution` tags.

Nit-pick: I think we could have left `%{short_name}` the way it was but it doesn't really matter.

Reply:
Updated